app/Resources/views/committee/show.html.twig

```diff
@@ -131,6 +133,16 @@
                 <img src="{{ asset('images/loader-sm.gif') }}" id="committee-follow-btn-loader" />
                 Suivre ce comité
             </button>
+        {% endif %}
+
+        {% if is_host %}
+            <button role="button"
+                    class="btn btn--no-border btn--text-blue committee-unfollow btn--no-wrap"
+                    id="committee-toggle-membership"
+                    disabled="disabled"
+                    title="En tant qu'animateur, vous ne pouvez pas cesser de suivre votre comité">
+                Quitter ce comité
+            </button>
```

Comment:
Can you also add the same kind of text than on line https://github.com/EnMarche/en-marche.fr/pull/742/files#diff-5f04b136c3d27995986e6b2fbc7da527R114 ?